Lib/test/test_timeout.py

```diff
@@ -180,19 +179,15 @@ def testConnectTimeout(self):
         # timeframe).
 
         # For the records, the whitehole/blackhole configuration has been set
-        # up using the 'pf' firewall (available on BSDs), using the following:
+        # up using the 'iptables' firewall, using the following rules:
         #
-        #   ext_if="bge0"
+        # -A INPUT -p tcp --destination-port 56666 -j DROP
+        # -A INPUT -p udp --destination-port 56666 -j DROP
+        # -A INPUT -p tcp --destination-port 56667 -j RETURN
+        # -A INPUT -p udp --destination-port 56667 -j RETURN
         #
-        #   blackhole_ip="35.8.247.6"
-        #   whitehole_ip="35.8.247.6"
-        #   blackhole_port="56666"
-        #   whitehole_port="56667"
-        #
-        #   block return in log quick on $ext_if proto { tcp udp } \
-        #       from any to $whitehole_ip port $whitehole_port
-        #   block drop in log quick on $ext_if proto { tcp udp } \
-        #       from any to $blackhole_ip port $blackhole_port
+        # See pillar/base/firewall/snakebite.sls in
+        # https://github.com/python/psf-salt for the current configuration.
```

Comment:
You may add a direct link:

```
# Link the the current configuration (2023):
# https://github.com/python/psf-salt/blob/main/pillar/base/firewall/snakebite.sls
```

Reply:
I avoided this because it would overflow the apparent 80 column standard in the file 😅.

Reply:
Will update.

Reply:
I'm unsure on if adding a date to the comment makes sense given that the link goes to the latest version.